recipe/meta.yaml

```diff
@@ -19,14 +19,13 @@ requirements:
   host:
     - python
     - pip
-    - jupyter-packaging >=0.9,<2
-    - jupyter_server
     - setuptools
```

Comment:
`setuptools` is no more needed as a build backend, see https://github.com/jupyterlab/jupyterlab_server/blob/ca2fc08221673126b30e73750ad9fc213daa5c50/pyproject.toml#L2-L3

Reply:
Removed `setuptools` and `wheels` from the recipe:
https://github.com/AnacondaRecipes/jupyterlab_server-feedstock/blob/2daa36896c45ed00ac4aa0a6ed344a5c7b5ec521/recipe/meta.yaml#L21&L22